dart/example/main.dart

```diff
@@ -82,6 +88,8 @@ Future<void> main() async {
   } finally {
     await Sentry.close();
   }
+
+  exit(0);
```

Comment:
do we need this if we `await` the events to be sent?

Reply:
yes, without it the dart process seems to never end

Reply:
Dart does not end the process or it gets a deadlock somewhere in our SDK? worth checking

Reply:
it seems to be caused by the `isolateErrorIntegration`

Reply:
✅ I removed the `exit` call

Reply:
but the problem still exists or what? just to know if you found out or not

Reply:
I found that it's caused by the isolateErrorIntegration, but didn't found how to fix it.

Reply:
task was not created, so I did myself
https://github.com/getsentry/sentry-dart/projects/1#card-50215663